terraform/outputs.tf

```diff
@@ -71,6 +71,11 @@ output "bigquery_unique_events_view_id" {
   value       = google_bigquery_table.unique_events_view.table_id
 }
 
+output "bigquery_commit_review_status_table_id" {
+  description = "BigQuery commit_review_status table resource."
+  value       = var.commit_review_status.enabled ? module.commit_review_status.commit_review_status_table_id : null
```

Comment:
You may need to add `commit_review_status_table_id` as an output to the commit_review_status module so that you can access it here

Reply:
@lock14 Did you address this comment? I dont see the output in the codebase...

https://github.com/search?q=repo%3Aabcxyz%2Fgithub-metrics-aggregator%20commit_review_status_table_id&type=code

Reply:
Just curious, if we are just outputting the input, why do this needle threading?

Reply:
Can we use the property from the bigqery table resource instead?

Reply:
@verbanicm that makes more sense. done.